packages/spectacular/src/lib/pipe-testing/pipe-harness/create-pipe-harness.ts

```diff
@@ -25,6 +29,17 @@ function createPipeFixture<TValue>(
   return pipeFixture;
 }
 
+function getPipeName(pipeType: Type<PipeTransform>): string {
```

Comment:
Nice! 👏
Does this also work if a pipe has an argument in its constructor?

Reply:
Verified! ✅

https://github.com/ngworker/ngworker/blob/spectacular/feat/remove-pipeName-option/packages/spectacular/src/lib/pipe-testing/metadata/get-pipe-annotation.spec.ts✅